bertran_nelson/005/actividad_5.py

```diff
@@ -0,0 +1,23 @@
+"""
+	Función que reciba como parametros dos números (a y b), y
+	devuelva una tupla con todos los números primos que se encuentra entre
+	a y b (incluyendo a y b si son primos).
+    Por defecto a=0, y b=100
+"""
+
+def primosRango(a=0, b=100):
+
+	primos = [2]
+
+	for num in range(2, b + 1):
+
+		for primo in primos:
+
+			if num % primo == 0:
+				break
+		else:
+			primos.append(num)
+
+	return tuple([p for p in primos if (p >= a and p <= b)])
```

Comment:
Podriamos evitar ese recorrido, si en el primer for probamos que números son primos desde a  hasta b, en lugar desde 2 hasta b.

Reply:
Gracias por la devolución profe, voy a tratar de mejorarlo.